meson.build

```diff
@@ -1,19 +1,65 @@
 project('hack-sound-server', version: '0.0.0')
 
+python = find_program('python3')
+pkgdatadir = join_paths(get_option('datadir'), meson.project_name())
+
 session_bus_services_dir = get_option('session-bus-services-dir')
 
 if session_bus_services_dir == ''
    session_bus_services_dir = join_paths(get_option('datadir'), 'dbus-1', 'services')
 endif
 
 install_data(
-    'com.endlessm.HackSoundServer.service',
+    join_paths('service', 'com.endlessm.HackSoundServer.service'),
     install_dir: session_bus_services_dir
 )
 
+conf = configuration_data()
+conf.set('DATADIR', join_paths(get_option('prefix'), get_option('datadir')))
+conf.set('LIBDIR', join_paths(get_option('prefix'), get_option('libdir')))
+conf.set('PACKAGE', meson.project_name())
+
+pkglibdir = join_paths(get_option('libdir'), meson.project_name())
+install_dir = join_paths(pkglibdir, 'python', 'hack_sound_server')
+
+run_command(
+    python, '-m', 'compileall',
+    join_paths(meson.current_source_dir(), 'hack_sound_server')
+)
+
+install_subdir(
+    'src',
+    exclude_files: 'configure.py.in',
+    install_dir: install_dir,
+    strip_directory: true
+)
+
+configure_file(
+    input: join_paths('src', 'configure.py.in'),
+    output: 'configure.py',
+    install: true,
+    install_dir: install_dir,
+    configuration: conf
+)
+
+bin = join_paths(meson.current_build_dir(), meson.project_name())
```

Comment:
I would not mix `meson.project_name()` with `hack-sound-server.in` in line 48. Either do all of them parametrized (and use the variable in line 49 as well), or all of them as `hack-sound-server`/`hack-sound-server.in`. Looking at the current state of things, it's probably better to just hardcode the name.

I don't think it's needed to join with `current_build_dir` here.